Vladislav_Sivtsov/05.Ansible.Start/README.md

```diff
@@ -0,0 +1,102 @@
+### Homework Assignment 1: Setting Up Ansible
+```bash
+ mkdir 05.Ansible.Start
+ 1128  ls
+ 1129  cd 05.Ansible.Start
+ 1130  sudo apt update
+ 1131  sudo apt upgrade
+ 1132  sudo apt install python3-pip
+ sudo apt update\nsudo apt install ansible
+  vim helloAnsible.yaml
+  ```yaml
+  
+    - name: Hello Ansible Playbook
+      hosts: localhost
+      connection: local
+      tasks:
+        - name: Print hello message
+          debug:
+            msg: "Hello, Ansible!"
+        
```

Comment:
you do not close code blocks, please check all, I see just open. It is not HTML, please close each of them